packages/cli/src/modules/chat-hub/chat-session.repository.ts

```diff
@@ -13,49 +13,66 @@ export class ChatHubSessionRepository extends Repository<ChatHubSession> {
 	}
 
 	async createChatSession(session: Partial<ChatHubSession>, trx?: EntityManager) {
-		return await withTransaction(this.manager, trx, async (em) => {
-			await em.insert(ChatHubSession, session);
-			return await em.findOneOrFail(ChatHubSession, {
-				where: { id: session.id },
-				relations: ['messages'],
-			});
-		});
+		return await withTransaction(
+			this.manager,
+			trx,
+			async (em) => {
+				await em.insert(ChatHubSession, session);
+				return await em.findOneOrFail(ChatHubSession, {
+					where: { id: session.id },
+					relations: ['messages'],
+				});
+			},
+			false,
+		);
 	}
 
-	async updateLastMessageAt(id: string, lastMessageAt: Date, trx?: EntityManager) {
-		return await withTransaction(this.manager, trx, async (em) => {
-			await em.update(ChatHubSession, { id }, { lastMessageAt });
-			return await em.findOneOrFail(ChatHubSession, {
-				where: { id },
-				relations: ['messages'],
-			});
-		});
+	async updateLastMessageAt(id: string, trx?: EntityManager) {
+		return await withTransaction(
+			this.manager,
+			trx,
+			async (em) => {
+				await em.update(ChatHubSession, { id }, { lastMessageAt: new Date() });
```

Comment:
Repository is persistence layer, isn't it better to pass `lastMessageAt` as an argument?

Reply:
I'm dropping this and `updateChatTitle`, moving towards just having `updateChatSession`

Reply:
Good idea

Reply:
Addressed in [7fc1f0a](https://github.com/n8n-io/n8n/pull/23397/commits/7fc1f0ac5f0252c1f10a8d411d30e1d5a5375ad3), together with fixing merge conflicts